fix: Task :acceptanceTest Fail

jenkins build 55

```
> Task :acceptanceTest

woowacrew.feed.controller.FeedApiControllerTest > updateFeedArticles() FAILED
    java.lang.AssertionError at FeedApiControllerTest.java:109
        Caused by: java.lang.AssertionError at FeedApiControllerTest.java:109

woowacrew.feed.controller.FeedApiControllerTest > jenkins_ip는_로그인_없이도_피드_업데이트_요청이_가능하다() FAILED
    java.lang.AssertionError at FeedApiControllerTest.java:184
        Caused by: java.lang.AssertionError at FeedApiControllerTest.java:184
```

--- a/src/main/java/woowacrew/security/AbstractSecurityConfig.java
+++ b/src/main/java/woowacrew/security/AbstractSecurityConfig.java
@@ -6,11 +6,13 @@ import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.builders.WebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
 import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+import org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter;
 import org.springframework.web.cors.CorsConfiguration;
 import org.springframework.web.cors.CorsConfigurationSource;
 import org.springframework.web.cors.CorsUtils;
 import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
 import woowacrew.security.filter.AuthorityUpdateFilter;
+import woowacrew.security.filter.AddressLoggingFilter;
 import woowacrew.security.filter.SocialLoginFilter;
 import woowacrew.security.handler.AccessDenyHandler;
 import woowacrew.security.handler.CustomLogoutSuccessHandler;
@@ -82,6 +84,7 @@ public abstract class AbstractSecurityConfig extends WebSecurityConfigurerAdapte
                 .logoutSuccessHandler(new CustomLogoutSuccessHandler());
         http
                 .addFilterBefore(socialLoginFilter(), UsernamePasswordAuthenticationFilter.class)
-                .addFilterBefore(authorityUpdateFilter(), UsernamePasswordAuthenticationFilter.class);
+                .addFilterBefore(authorityUpdateFilter(), UsernamePasswordAuthenticationFilter.class)
+                .addFilterBefore(new AddressLoggingFilter(), WebAsyncManagerIntegrationFilter.class);
     }
 }

--- a/src/main/java/woowacrew/security/filter/AddressLoggingFilter.java
+++ b/src/main/java/woowacrew/security/filter/AddressLoggingFilter.java
@@ -1,0 +1,21 @@
+package woowacrew.security.filter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.filter.GenericFilterBean;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import java.io.IOException;
+
+public class AddressLoggingFilter extends GenericFilterBean {
+    private static final Logger log = LoggerFactory.getLogger(AddressLoggingFilter.class);
+
+    @Override
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
+        log.info("Request Ip : {}, {}", request.getLocalAddr(), request.getRemoteAddr());
+        chain.doFilter(request,response);
+    }
+}

--- a/src/main/resources/application.yml
+++ b/src/main/resources/application.yml
@@ -4,10 +4,6 @@ spring:
   thymeleaf:
     prefix: classpath:/static/
 
-jenkins:
-  ipv4: 127.0.0.1
-  ipv6: 0:0:0:0:0:0:0:1 #localhost
-
 ---
 
 spring:
@@ -29,6 +25,9 @@ spring:
     console:
       enabled: true
 
+jenkins:
+  ipv4: 127.0.0.1
+  ipv6: 0:0:0:0:0:0:0:1 #localhost
 
 logging:
   config: "classpath:logback-dev.xml"

